Dockerfile.telco-core

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal
+FROM registry.redhat.io/ubi9/ubi-minimal:9.5
```

Comment:
The container image should be based on the same RHEL version as OCP, I guess it makes sense to use an argument like `RHEL_VERSION` to be able to override in case needed. Please see https://github.com/openshift-kni/dpdk-base/pull/5.

OCP 4.19 GA will be based on RHEL 9.4 EUS so perhaps we should use RHEL version as 9.4 here?